test/spec/controllers/tasksCtrlSpec.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 describe('Tasks Controller', function() {
-  var $rootScope, shared, scope, user, ctrl;
+  var $rootScope, shared, scope, user, User, ctrl;
 
   beforeEach(function() {
-    user = specHelper.newUser();
```

Comment:
You can leave this and just have:

```
User = {
  user: user
};
```

Then in the tests you can reference `user` instead of `User.user` for everything.